.rubocop_todo.yml

```diff
@@ -1,154 +1,106 @@
-require: rubocop-rspec
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2017-06-22 15:12:41 -0700 using RuboCop version 0.49.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
 
-Security/MarshalLoad:
-  Exclude:
-    - 'app/models/concerns/hyrax/user.rb'
-
-Metrics/ClassLength:
-  Exclude:
-    - 'app/actors/hyrax/actors/file_set_actor.rb'
-    - 'app/controllers/hyrax/file_sets_controller.rb'
-    - 'app/forms/hyrax/forms/permission_template_form.rb'
-    - 'app/presenters/hyrax/work_show_presenter.rb'
-    - 'app/services/hyrax/user_stat_importer.rb'
-    - 'lib/generators/hyrax/templates/catalog_controller.rb'
-    - 'lib/generators/hyrax/install_generator.rb'
-    - 'lib/hyrax/configuration.rb'
-
-Metrics/ParameterLists:
+# Offense count: 1
+Lint/ScriptPermission:
   Exclude:
-    - 'app/jobs/batch_create_job.rb'
+    - 'Rakefile'
 
+# Offense count: 104
 Metrics/AbcSize:
-  Exclude:
-    - 'app/indexers/hyrax/file_set_indexer.rb'
-    - 'app/services/hyrax/workflow/permission_query.rb'
+  Max: 58
 
-Metrics/ModuleLength:
-  Exclude:
-    - 'app/controllers/concerns/hyrax/collections_controller_behavior.rb'
-    - 'app/controllers/concerns/hyrax/works_controller_behavior.rb'
-    - 'app/helpers/hyrax/hyrax_helper_behavior.rb'
-    - 'app/models/concerns/hyrax/ability.rb'
-    - 'app/services/hyrax/workflow/permission_query.rb'
-    - 'spec/services/hyrax/workflow/permission_query_spec.rb'
+# Offense count: 8
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 278
 
+# Offense count: 79
+# Configuration parameters: CountComments.
 Metrics/MethodLength:
-  Exclude:
-    - 'app/controllers/concerns/hyrax/api.rb'
-    - 'app/controllers/concerns/hyrax/controller.rb'
-    - 'app/controllers/hyrax/my_controller.rb'
-    - 'app/controllers/concerns/hyrax/works_controller_behavior.rb'
-    - 'app/indexers/hyrax/file_set_indexer.rb'
-    - 'app/inputs/multi_value_select_input.rb'
-    - 'app/models/concerns/hyrax/solr_document/export.rb'
-    - 'app/services/hyrax/workflow/permission_query.rb'
-    - 'lib/generators/hyrax/install_generator.rb'
-    - 'lib/hyrax/rails/routes.rb'
-    - 'spec/support/**/*'
+  Max: 51
 
-Style/LineEndConcatenation:
-  Exclude:
-    - 'spec/test_app_templates/lib/generators/**/*'
-    - 'lib/generators/**/*'
+# Offense count: 6
+# Configuration parameters: CountComments.
+Metrics/ModuleLength:
+  Max: 230
 
-Style/PredicateName:
-  Exclude:
-    - 'app/helpers/hyrax/collections_helper.rb'
+# Offense count: 2
+# Configuration parameters: CountKeywordArgs.
+Metrics/ParameterLists:
+  Max: 6
 
-Rails/Output:
-  Exclude:
-    - 'lib/generators/**/*'
+# Offense count: 37
+RSpec/AnyInstance:
+  Enabled: false
```

Comment:
I'd prefer we enable this and work towards removing those exceptions we've already granted.